assets/javascripts/discourse-assign/controllers/group-assignments.js.es6

```diff
@@ -1,31 +1,33 @@
 import { inject as service } from "@ember/service";
 import Controller, { inject as controller } from "@ember/controller";
+import { ajax } from "discourse/lib/ajax";
 
 export default Controller.extend({
   router: service(),
   application: controller(),
   loading: false,
+  offset: 0,
 
   findMembers(refresh) {
-    if (this.loading || !this.model) {
+    if (refresh) {
+      this.set("members", this.model.members);
       return;
     }
 
-    if (!refresh && this.model.members.length >= this.model.user_count) {
-      this.set("application.showFooter", true);
+    if (this.loading || !this.model) {
       return;
     }
 
-    this.set("loading", true);
-    this.model
-      .findMembers({ order: "", asc: true, filter: null }, refresh)
-      .finally(() => {
-        this.setProperties({
-          "application.showFooter":
-            this.model.members.length >= this.model.user_count,
-          loading: false
-        });
-      });
+    if (this.model.members.length >= this.offset + 50) {
+      this.set("loading", true);
+      this.set("offset", this.offset + 50);
+      ajax(`/assign/members/${this.groupName}?offset=${this.offset}`).then(
+        result => {
+          this.set("members", this.members.concat(result.members));
```

Comment:
I think `members` is an ember [MutableArray](https://api.emberjs.com/ember/2.15/classes/Ember.MutableArray/methods/pushObject?anchor=pushObjects)? So we should be able to use `pushObjects`
```suggestion
          this.members.pushObjects(result.members);
```
I think that should make it more efficient for ember to re-render the template (since the array isn't being completely replaced)

Reply:
Sure, will change this in a moment

Reply:
added in the latest commit